### ISSUE TITLE
DID Method Specification Requirements

### ISSUE THREAD

**Issue by nadalin** [w3c-ccg/did-wg-charter#64]
"Recommend a set of requirements for DID Method specifications that are conformant with the data model and syntax(es). The DID Method specification authoring requirements will recommend a list of mandatory and optional operations, with associated descriptions, which are expected to be defined in DID method specifications. "

Since this specification is not defining DID methods I would think that the its out of scope to define requirements for something this WG is not doing. If ever a new WG is to take on DID Methods this WG would define the requirements.

**Comment by talltree** [comment]
I strongly disagree. DID Method specs are not necessarily going to be done in the W3C or in any SDO. The model here is the URI spec, RFC 3986. It imposes requirements on compliant URI scheme specs. We're doing the same thing.

**Comment by nadalin** [comment]
@talltree I suggest adding a Generic DID Method Syntax specification to the deliverables section then as this should not be part of the Decentralized Identifiers v1.0 specification so this can be changed independently if needed.

The wording also suggests that this is more than syntax which is a concern
